backend/app/validators/project_geometry_validator.rb

```diff
@@ -27,8 +27,7 @@ def decode_geo_json
   def validation_of(geometry)
     return if geometry.blank?
 
-    center = GeoJsons::ToCentroid.new(geometry).call
-    @project.latitude, @project.longitude = [center.y, center.x]
+    @project.center = GeoJsons::ToCentroid.new(geometry).call
```

Comment:
I think it would be better to name it `centroid`, because that is the common name we always use. Also, `project.center` does not say much, better `project.centroid` is more descriptive.

Reply:
No problem, I will rename it ;)